### ISSUE TITLE
Impossible to invoke a method ("getCpEditUrl") on a null variable

### ISSUE THREAD

**Issue by CMeldgaard** [verbb/gift-voucher#108]
### Describe the bug

If a gift voucher has been created through an order, and the order is the deleted (since this is the default approach to "cancel" an order), and you then go into the voucher code, it will give this error, since it has a orderId, but the order doesn't exists.

What approach to fix it is correct, i can't see. Because on one hand, if the order gets deleted it means the purchase has been cancelled - shouldn't the voucher code then also be deleted, to avoid double maintenance of order/vouchers?

### Steps to reproduce

1. Buy a voucher
2. Delete the order
3. Try to edit the voucher code.

### Craft CMS version

4.x

### Plugin version

3.0.4

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/gift-voucher` requirement in `composer.json` to:

```
"require": {
  "verbb/gift-voucher": "dev-craft-4 as 3.0.5",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [3.0.6](https://github.com/verbb/gift-voucher/releases/tag/3.0.6)
